onnxruntime/server/grpc/grpc_app.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+#include <grpcpp/grpcpp.h>
+#include "prediction_service_impl.h"
+#include "../environment.h"
+
+namespace onnxruntime {
+namespace server {
+class GRPCApp {
+ public:
+  GRPCApp(const std::shared_ptr<onnxruntime::server::ServerEnvironment>& env, std::string host, const unsigned short port);
+  ~GRPCApp() = default;
+  GRPCApp(GRPCApp const& other) = delete;
+  GRPCApp(GRPCApp&& other) = default;
+
+  //Block until the server shuts down.
+  void Run();
+
+ private:
+  grpc::PredictionServiceImpl m_service;
```

Comment:
please follow ORT coding conventions. service_.

Reply:
can you link me to the naming?